base/broadcast.jl

```diff
@@ -219,6 +219,12 @@ argtype(bc::Broadcasted) = argtype(typeof(bc))
 _eachindex(t::Tuple{Any}) = t[1]
 _eachindex(t::Tuple) = CartesianIndices(t)
 
+Base.IndexStyle(bc::Broadcasted) = IndexStyle(typeof(bc))
+Base.IndexStyle(::Type{<:Broadcasted{<:Any,Tuple{Axis}}}) where {Axis} = IndexStyle(Axis)
```

Comment:
```suggestion
Base.IndexStyle(::Type{<:Broadcasted{<:Any,Tuple{Any}}}) = IndexLinear()
```

It doesn't really matter how the particular axis is implemented — we can still just use one linear index to access into it.